src/init/features/dataconnect/sdk.ts

```diff
@@ -75,11 +76,13 @@ async function askQuestions(setup: Setup, config: Config): Promise<SDKInfo> {
   } else {
     // If we aren't, ask the user where their app is, and try to autodetect from there
     logBullet(`Couldn't automatically detect your app directory.`);
-    appDir = await promptForDirectory({
-      config,
-      message:
-        "Where is your app directory? Leave blank to set up a generated SDK in your current directory.",
-    });
+    appDir =
```

Comment:
Should this happen before line 71?

If `FDC_APP_FOLDER` is specified, but `process.cwd()` is detected to be a iOS app, it should still respect `FDC_APP_FOLDER`.

Reply:
Good call, fixed.